core/nwb.ophys.yaml

```diff
@@ -1,4 +1,55 @@
 groups:
+- neurodata_type_def: OnePhotonSeries
+  neurodata_type_inc: ImageSeries
+  doc: Image stack recorded over time from 1-photon microscope.
+  attributes:
+  - name: pmt_gain
+    dtype: float32
+    doc: Photomultiplier gain.
+    required: false
+  - name: scan_line_rate
+    dtype: float32
+    doc: Lines imaged per second. This is also stored in /general/optophysiology but
+      is kept here as it is useful information for analysis, and so good to be stored
+      w/ the actual data.
+    required: false
+  - name: exposure_time
+    dtype: float32
+    doc: Exposure time of the sample; often the inverse of the frequency.
+    required: false
+  - name: binning
+    dtype: uint8
+    doc: Amount of pixels combined into 'bins'; could be 1, 2, 4, 8, etc.
+    required: false
+  - name: power
+    dtype: float32
+    doc: Power of the excitation in mW, if known.
+    required: false
+  - name: intensity
+    dtype: float32
+    doc: Intensity of the excitation in mW/mm^2, if known.
+    required: false
+  datasets:
+  - name: field_of_view
+    dtype: float32
+    dims:
+    - - width|height
+    - - width|height|depth
+    shape:
+    - - 2
+    - - 3
+    doc: Width, height and depth of image or imaged area, in meters.
+    quantity: '?'
+    attributes:
+    - name: unit
+      dtype: text
+      value: meters
+      doc: Unit of measurement for the width, height, and depth of the image or imaged area.
```

Comment:
It seems like this is handled by `ImagingPlane.manifold` and `ImagingPlane.origin_coords`, no?

Reply:
Hmm.. This field was copied over from the `TwoPhotonSeries`, where it's also defined.

Note that `manifold` is deprecated in favor of `origin_coords` + `grid_spacing`: https://github.com/catalystneuro/nwb-schema/blob/add_one_photon_series/core/nwb.ophys.yaml#L262-L263

As for the difference with the `origin_coords` + `grid_spacing` - my best guess here is maybe you could have several `PhotonSeries` corresponding to the same general `ImagingPlane` or `OpticChannels` but maybe the field of view for each series spans only a subset of the entire plane?

If we can remember why it was designed that way for the `TwoPhotonSeries` case maybe it will help make more sense here, but maybe it should also just be a thing defined only for the `ImagingPlane` as well, not sure...

Reply:
@oruebel, @rly , any ideas about this?